crypto/asn1/asn_mstbl.c

```diff
@@ -98,7 +100,8 @@ static int do_tcreate(const char *value, const char *name)
     if (rv == 0) {
         if (cnf)
             ERR_raise_data(ERR_LIB_ASN1, ASN1_R_INVALID_STRING_TABLE_VALUE,
-                           "field=%s, value=%s", cnf->name, cnf->value);
+                           "field=%s, value=%s", cnf->name,
+                                                 cnf->value ? cnf->value : value);
```

Comment:
This should really be `cnf->value != NULL` as per our style

Reply:
fixed